src/components/sections/SchoolPage/SchoolHeader.tsx

```diff
@@ -51,13 +53,19 @@ const SchoolHeader: FC<SchoolHeaderProps> = ({
       <span className="public">{isPublic ? 'szkoła publiczna' : 'szkoła niepubliczna'}</span>
       <h1>{!isLoading && schoolName}</h1>
       <span className="district">{!isLoading && district}</span>
-      <p>
-        [Opis szkoły: np.] Publiczne liceum ogólnokształcące w Warszawie założone w 1874. Jest
-        najstarszym warszawskim liceum.
-      </p>
       <AddRemoveFavourite isFavourite={isFavourite} onClick={toggleFavourite} />
     </HeaderWrapper>
   );
 };
 
 export default SchoolHeader;
+
+export const SCHOOL_HEADER_PROPS_FRAGMENT = gql`
```

Comment:
I think this should be in another file (same with others graphql definitions, that are places in files with components).